pkg/controller/quota/quota.go

```diff
@@ -113,8 +119,10 @@ func addCompute(containers map[string]apiv1.Container, appInstance *apiv1.AppIns
 			requirements = all.Requirements
 		}
 
-		quotaRequest.Spec.Resources.CPU.Add(requirements.Requests["cpu"])
-		quotaRequest.Spec.Resources.Memory.Add(requirements.Requests["memory"])
+		cpu := quotaRequest.Spec.Resources.Quantities[adminv1.CPU]
+		cpu.Add(requirements.Requests["cpu"])
```

Comment:
I assume you broke this up because the compiler was giving an error when it was combined.

If that is the case, then `cpu` needs to be put back in the map after it is modified.

Same for the others below, and the ones that @g-linville called out.

Reply:
How would you and @g-linville feel about making the Quantities in the map be pointers? This would make it so that the mutating function called to resources.Quantity types would persist.

Reply:
That sounds good to me.

Reply:
As I started implementing this doing nil checks everywhere got to be more of a hassle so I just ended up add it back onto the map. like you originally suggested @g-linville.